docs/sources/panels/panel-library.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "Library panels"
+weight = 400
++++
+
+# Library panel
+
+Library panels allow users to create reusable panels that aren’t tied to any one dashboard. As a result, any changes made to one instance of the library panel affecting all other instances.
```

Comment:
Suggestion from the word doc. My goal was to reframe the value statement from what they aren't - tied to one dashboard - to the value they offer: reuse:

```Library panels allow users to create reusable panels where any changes made to one instance of the library panel is reflected on every dashboard affecting all other instances. where the panel is used. These panels can be saved in folders alongside Dashboards and streamline reuse of panels across multiple dashboards.
```

Reply:
Added, thank you!